scripts/rosdistro_build_cache

```diff
@@ -67,6 +67,9 @@ def parse_args(args=sys.argv[1:]):
     add('--ignore-local', action='store_true', default=False,
         help='Ignore locally available cache'
     )
+    add('--include-source', action='store_true', default=False,
+        help='Also include source branch package XMLs in the cache'
+    )
```

Comment:
For now, I'm implemented this as an option rather than a separate script. I want to understand the file size impact once deduplication is done, and if it's still really bad, then we can revisit making a separate cache file, separate script, etc.